assets/templates/partials/header.tmpl

```diff
@@ -270,6 +270,6 @@
         </div>
     {{end}}
     {{ if and .BetaBannerEnabled .FeatureFlags.SixteensVersion }}
-        {{ template "partials/banners/cmd-beta" . }}
+        {{ template "partials/banners/search-beta" . }}
```

Comment:
The `search-beta` template isn't in source control. Do you need to push the file rename?